lib/admin/class-groups-admin-users.php

```diff
@@ -222,9 +222,9 @@ public static function views_users( $views ) {
 				esc_attr( __( 'Choose groups &hellip;', 'groups' ) )
 			);
 			$user_group_table = _groups_get_tablename( 'user_group' );
-			$groups = Groups_Group::get_groups( array( 'order_by' => 'name', 'order' => 'ASC' ) );
+			$groups = apply_filters( 'groups_view_users', Groups_Group::get_groups( array( 'order_by' => 'name', 'order' => 'ASC' ) ) );
```

Comment:
We need to review this filter `groups_view_users` because it does not adhere to our naming conventions, noted for review and inclusion after adjustment.